Source/Content.swift

```diff
@@ -14,9 +18,10 @@ public class Content: NSObject {
   public private(set) var initialPosition: Position
   private let group = ConstraintGroup()
 
-  public init(view: UIView, position: Position) {
+  public init(view: UIView, position: Position, centered: Bool = true) {
```

Comment:
Your variable stars as `true`, any reason why you need both the default value here and the declaration above?